src/components/Vuetify/mixins/lang.ts

```diff
@@ -29,6 +29,9 @@ export default function lang (config: Options['lang'] = {}): VuetifyLanguage {
     locales: Object.assign({ en }, config.locales),
     current: config.current || 'en',
     t (key, ...params) {
+      if (typeof key === 'number') {
```

Comment:
Should be added to the public type definitions, then you can remove `as number` because it'll be narrowed.

Reply:
> Should be added to the public type definitions

Will do &#128077;